agent/tcs/handler/handler.go

```diff
@@ -145,6 +147,15 @@ func ackPublishMetricHandler(timer *time.Timer) func(*ecstcs.AckPublishMetric) {
 	}
 }
 
+// ackPublishHealthMetricHandler consumes the ack message from backend. The backend sends
+// the ack each time it processes a health message
+func ackPublishHealthMetricHandler(timer *time.Timer) func(*ecstcs.AckPublishHealth) {
+	return func(*ecstcs.AckPublishHealth) {
+		seelog.Debug("Received ACKPublishHealth from tcs")
```

Comment:
Can you check if the message id can be logged here? That'd improve the message tracking if we need to debug anything with lossy messages in the future.

Reply:
The message ID isn't available alone here, we can only log the whole message.

Reply:
oh, never mind then